deploy/helm/grafana-operator/values.yaml

```diff
@@ -100,9 +114,9 @@ tolerations: []
 # -- pod affinity
 affinity: {}
 
-# -- Enable this to use with Prometheus Operator
 serviceMonitor:
-  # -- When set true then use a ServiceMonitor to configure scraping
+  # -- Used with Prometheus Operator
+  # When true Create a ServiceMonitor to enable metric scraping
```

Comment:
Same as in other places - the lines get merged in docs, so better to write in a way that takes that into account.
I think it was useful to remove `# -- Enable this to use with Prometheus Operator` as it was generating a cluttered line in README.md. For the rest, let's use something like `# -- Whether to create a ServiceMonitor`. - Everyone knows what a `ServiceMonitor` is and where it comes from, so we don't need to explain much.